lib/models/usuario.js

```diff
@@ -159,6 +164,20 @@ export default class Usuario extends Model {
     return !isEmpty(turnos);
   }
 
+  generarTokenTemporal() {
+    return generarJWT(this.id, this.dni, '15m');
+  }
+
+  async actualizarContrasenia(contrasenia) {
+    await this.update({
+      contrasenia: bcrypt.hashSync(contrasenia, bcrypt.genSaltSync()),
+    });
```

Comment:
Siguiendo lo que decía antes, acá quedaría:

```suggestion
    await this.update({
      contrasenia: bcrypt.hashSync(contrasenia, bcrypt.genSaltSync()),
      tokenTemporal: null
    });
```